Common subdirectories: jnos/backup and jnos2/backup
Common subdirectories: jnos/installerv2.1 and jnos2/installerv2.1
Common subdirectories: jnos/j2pwmgr and jnos2/j2pwmgr
Common subdirectories: jnos/k6fsh and jnos2/k6fsh
diff -u jnos/mailbox.c jnos2/mailbox.c
--- jnos/mailbox.c	2016-07-03 10:04:21.000000000 -0400
+++ jnos2/mailbox.c	2016-09-13 22:45:14.293150509 -0400
@@ -1646,7 +1646,7 @@
   
     /* Now say goodbye */
     if(!(m->privs & IS_EXPERT))
-        tprintf("\nThank you %s, for calling %s JNOS.\n",
+        tprintf("%s, thank you for calling %s. Come back soon.\nRemember to keep packet alive!\n",
 #ifdef USERLOG
         m->username ? m->username : m->name,
 #else
diff -u jnos/makefile jnos2/makefile
--- jnos/makefile	2015-12-08 12:37:21.000000000 -0500
+++ jnos2/makefile	2016-09-14 18:18:30.268383703 -0400
@@ -198,7 +198,8 @@
 	echo "YOU MUST EDIT scanjmsg.c TO SPECIFY WHERE scanjmsg.dat IS LOCATED"
 
 clean:
-	$(RM) *.[oa]
+#	Oops, forgot to remove possible existing binaries too! - N1URO
+	$(RM) *.[oa] jnos jnospwmgr jnosinstaller j2pwmgr/j2pwrtns.o
 
 clients.a: $(CLIENTS)
 	$(RM) clients.a
diff -u jnos/mboxcmd.c jnos2/mboxcmd.c
--- jnos/mboxcmd.c	2016-07-03 10:04:26.000000000 -0400
+++ jnos2/mboxcmd.c	2016-09-13 22:45:14.293150509 -0400
@@ -478,7 +478,7 @@
 #ifdef NETROM
     if(Nr_iface != NULLIF) { /* Use netrom call, and alias (if exists) */
         if(*Myalias != '\0')
-            sprintf(Mbnrid,"%s:%s ",pax25(tmp,Myalias),
+            sprintf(Mbnrid,"%s:%s} ",pax25(tmp,Myalias),
             pax25(tmp2,Nr_iface->hwaddr));
         else
             sprintf(Mbnrid,"%s ",pax25(tmp,Nr_iface->hwaddr));
@@ -486,7 +486,7 @@
     }
     /* Use Mycall, and alias (if exists) */
     if(*Myalias != '\0')
-        sprintf(Mbnrid,"%s:%s ",pax25(tmp,Myalias),pax25(tmp2,Mycall));
+        sprintf(Mbnrid,"%s:%s} ",pax25(tmp,Myalias),pax25(tmp2,Mycall));
     else
 #endif
 #ifdef AX25
diff -u jnos/misc.c jnos2/misc.c
--- jnos/misc.c	2007-04-12 14:08:19.000000000 -0400
+++ jnos2/misc.c	2016-09-13 22:45:14.293150509 -0400
@@ -14,6 +14,8 @@
         n = strlen(s);
         if(!strncmp(s,"convers",n))
             p = IPPORT_CONVERS;
+	else if(!strncmp(s,"node",n))
+	    p = IPPORT_NODE;		/* URONode port - N1URO */
         else if(!strncmp(s,"telnet",n))
             p = IPPORT_TELNET;
         else if(!strncmp(s,"ttylink",n))
diff -u jnos/netuser.c jnos2/netuser.c
--- jnos/netuser.c	2007-04-12 14:08:19.000000000 -0400
+++ jnos2/netuser.c	2016-09-13 22:45:14.293150509 -0400
@@ -164,9 +164,12 @@
         case 1234:              /* Pulled out of the air */
             sprintf(port,"remote");
             break;
-        case 3600:
+        case 3600:		/* Convers server port */
             sprintf(port,"convers");
             break;
+	case 3694:		/* URONode inbound tcp port - N1URO*/
+	    sprintf(port,"node");
+	    break;
         default:
             sprintf(port,"%u",s->port);
             break;
diff -u jnos/nrcmd.c jnos2/nrcmd.c
--- jnos/nrcmd.c	2012-03-20 12:30:29.000000000 -0400
+++ jnos2/nrcmd.c	2016-09-13 22:45:14.293150509 -0400
@@ -498,7 +498,7 @@
                      * ALSO change the size of the STRSIZE define above !!!
                      */
                     sprintf(&temp[k],"%-16s %s %3d %3d %-8s %-9s %c %-5u",buf,
-                    rp->flags & G8BPQ_NODEMASK ? "(BPQ)" : "     ",
+                    rp->flags & G8BPQ_NODEMASK ? "     " : "     ",
                     bp->quality,bp->obsocnt,
                     np->iface->name,
                     pax25(neighbor,np->call),
@@ -631,7 +631,7 @@
 #endif
             ,buf,pax25(destbuf,rp->call)
 #ifdef G8BPQ
-            ,rp->flags & G8BPQ_NODEMASK ? "(BPQ)" : "     "
+            ,rp->flags & G8BPQ_NODEMASK ? "     " : "     "
 #endif
             );
         } else
@@ -648,7 +648,7 @@
 #endif
 #ifdef G8BPQ
 		if (npp && (npp->flags & G8BPQ_NODEMASK))
-			tprintf ("(BPQ)");
+			tprintf ("     ");
 		else
 #endif
 
@@ -793,7 +793,7 @@
 #endif
 #ifdef G8BPQ
 			if (rp && (rp->flags & G8BPQ_NODEMASK))
-				tprintf ("(BPQ)");
+				tprintf ("     ");
 			else
 #endif
 
Common subdirectories: jnos/old_inp_code and jnos2/old_inp_code
diff -u jnos/socket.h jnos2/socket.h
--- jnos/socket.h	2016-06-26 13:27:07.000000000 -0400
+++ jnos2/socket.h	2016-09-14 10:20:04.532733148 -0400
@@ -56,6 +56,7 @@
 #define IPPORT_RSYSOP 1513      /* Remote sysop login (totally arbitrary #) */
 #define IPPORT_CONVERS 3600     /* Converse */
 #define IPPORT_XCONVERS 3601    /* LZW Convers */
+#define IPPORT_NODE 3694	/* URONode telnet port - N1URO*/
 #define IPPORT_CALLDB 1235      /* Pulled out of the air GRACILIS */
 #define IPPORT_TRACE 1236       /* Pulled out of the air - WG7J */
 #define IPPORT_TERM 5000        /* Serial interface server port */
